packages/card/Card.test.tsx

```diff
@@ -1,17 +1,26 @@
-import { Card } from './Card';
-
+import * as expect from 'expect';
 import { h, mount } from 'bore';
+import { Card } from './index';
 
 describe( Card.is, () => {
 
-  it( 'should render', () => {
+  describe( `Custom element`, () => {
 
-    console.warn( 'Missing tests for Card!' );
+    it( `should be registered`, () => {
+      expect( customElements.get( Card.is ) ).toBe( Card );
+    } );
 
-    return mount(
-      <Card />
-    ).wait();
+    it( `should render via JSX IntrinsicElement`, () => {
+      return mount( <bl-card>Content</bl-card> ).wait(( element ) => {
+        expect( element.node.localName ).toBe( Card.is );
+      } );
+    } );
 
+    it( `should render`, () => {
+      return mount( <Card>Content</Card> ).wait(( element ) => {
+        expect( element.has( '.c-card' ) ).toBe( true );
+      } );
+    } );
   } );
 
```

Comment:
We cant do this, because we are using polyfill in firefox and safari where slot does not exist after render :(.